prov/ucx/src/ucx_av.c

```diff
@@ -269,6 +245,11 @@ static struct fi_ops_av ucx_av_ops = {
 	.lookup = ucx_av_lookup,
 };
 
+static void ucx_init_array(struct ofi_dyn_arr *arr, void *item)
+{
+	memset(item, 0, arr->item_size);
+}
```

Comment:
The array uses calloc to initialize entries to 0.  We can document that behavior.